device/globals/game.gd

```diff
@@ -26,14 +26,44 @@ var tooltip
 func set_mode(p_mode):
 	mode = p_mode
 
+func tooltip_check_clamp(tt_pos):
```

Comment:
Same. Might not need calling if there's no tooltip either.

Reply:
Let's go with `tooltip_clamped_position`.